Cargo.lock

```diff
@@ -5127,7 +5166,7 @@ dependencies = [
  "log",
  "murmur3",
  "once_cell",
- "opendal",
+ "opendal 0.44.2",
```

Comment:
Icelake is still on `0.44`. 😕

Reply:
checking

Reply:
I think it may be tolerated that the opendal version icelake depends on is inconsistent with risingwave. Icelake is still under rapid development, and we rely on one of its specific git commit instead of release version.

Reply:
It seems all dependabot PRs are blocked because it bumps icelake's dependency to 0.45.1 but our connector crate is still using 0.44.